languages/ruby/spec/oso/polar/polar_spec.rb

```diff
@@ -901,7 +901,6 @@ def self.test
       subject.register_class(RolesHelpers::Issue, name: 'Issue')
       subject.register_class(RolesHelpers::User, name: 'User')
       subject.load_file(roles_file)
-      subject.enable_roles
```

Comment:
How is the rest of this test still valid?

Reply:
I updated the 1 giant Polar Roles test in each language to the new syntax